src/wp-content/themes/twentyseventeen/functions.php

```diff
@@ -708,6 +708,14 @@ function wp_get_list_item_separator() {
  */
 function twentyseventeen_should_show_featured_image() {
 	$show_featured_image = ( is_single() || ( is_page() && ! twentyseventeen_is_frontpage() ) ) && has_post_thumbnail( get_queried_object_id() );
+	/**
+	 * Filters whether to show the featured image below the header on single posts and pages, unless the page is the front page.
+	 *
+	 * @param bool $show_featured_image Whether to the featured image below the header on single posts and pages.
+	 *
+	 * @since 3.7.0
```

Comment:
@pmbaldha It should come before `@param` as per WordPress doc. https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters